teamcity: failed test: version-upgrade

The following tests appear to have failed on master (roachtest): acceptance/version-upgrade

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+version-upgrade).

[#1474481](https://teamcity.cockroachdb.com/viewLog.html?buildId=1474481):


```
acceptance/version-upgrade
--- FAIL: roachtest/acceptance/version-upgrade (185.948s)
test artifacts and logs in: artifacts/acceptance/version-upgrade/run_1
	upgrade.go:525,upgrade.go:582,acceptance.go:69,test_runner.go:688: pq: value differs between gossip (2.1) and KV (2.0); try again later (<nil> after 1m58.924937708s)

test artifacts and logs in: artifacts/acceptance/version-upgrade/run_1
	upgrade.go:525,upgrade.go:582,acceptance.go:69,test_runner.go:688: pq: value differs between gossip (2.1) and KV (2.0); try again later (<nil> after 1m58.924937708s)





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cmd/roachtest/cluster.go
+++ b/pkg/cmd/roachtest/cluster.go
@@ -1534,7 +1534,7 @@ func (c *cluster) PutE(ctx context.Context, l *logger, src, dest string, opts ..
 
 	err := execCmd(ctx, c.l, roachprod, "put", c.makeNodes(opts...), src, dest)
 	if err != nil {
-		return errors.Wrap(ctx.Err(), "cluster.Put")
+		return errors.Wrap(err, "cluster.Put")
 	}
 	return nil
 }

